app/javascript/packs/rating.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function(){
+  $('.ratingW').on('DOMSubtreeModified', function(){
+          $(".toast").toast({ delay: 1500 });
+          $(".toast").toast('show');
+  });
+});
+
+function ratingStar(star){
```

Comment:
có thể không cần khai báo function như này nhé
```suggestion
$(document).on("click", ".ratingW", function(){
```

Reply:
E sửa như này bị lỗi á c